extensions/ql-vscode/src/local-databases-ui.ts

```diff
@@ -410,6 +411,23 @@ export class DatabaseUI extends DisposableObject {
     );
   };
 
+  private handleTourDependencies = async (): Promise<void> => {
+    // find workspace root folder
+    if (!workspace.workspaceFolders?.length) {
+      throw new Error("No workspace folder is open.");
+    } else {
+      // find tutorial queries folder
+      const tutorialQueriesPath = Uri.parse(
+        `${workspace.workspaceFolders?.[0].uri}/tutorial-queries`,
+      );
+      const cli = this.queryServer?.cliServer;
+      if (!cli) {
+        throw new Error("No CLI server found");
+      }
+      await cli.packInstall(tutorialQueriesPath.fsPath);
```

Comment:
And if you do above, you can change to this:

```suggestion
      await cli.packInstall(tutorialQueriesPath);
```